lib/commands/general.js

```diff
@@ -101,14 +101,13 @@ commands.toggleEnrollTouchId = async function (enabled = true) {
   await this.opts.device.enrollBiometric(enabled);
 };
 
+// memoized in constructor
 commands.getWindowSize = async function (windowHandle = 'current') {
```

Comment:
will memoizing this in the constructor do what we expect? what if the value of `this.isWebContext` changes within the course of a session?